cluster/gce/upgrade.sh

```diff
@@ -101,6 +102,48 @@ function upgrade-master() {
   wait-for-master
 }
 
+# TODO(mikedanese): delete when we don't support < 1.6
+function backfile-kubeletauth-certs() {
+  if [[ ! -z "${KUBEAPISERVER_CERT_BASE64:-}" && ! -z "${KUBEAPISERVER_CERT_BASE64:-}" ]]; then
+    return 0
+  fi
+
+  mkdir -p "${KUBE_TEMP}/pki"
+  echo "${CA_KEY_BASE64}" | base64 -d > "${KUBE_TEMP}/pki/ca.key"
+  echo "${CA_CERT_BASE64}" | base64 -d > "${KUBE_TEMP}/pki/ca.crt"
```

Comment:
These files are generated on the work station running kube-up.sh, and ${KUBE_TEMP} is deleted in the trap EXIT handler. We don't do any umasking elsewhere.

https://github.com/kubernetes/kubernetes/blob/master/cluster/common.sh#L987-L996